src/Sulu/Bundle/ContentBundle/Resources/js/views/PageEdit/PageEdit.js

```diff
@@ -0,0 +1,166 @@
+        }
+
+        WebspaceStore.loadWebspace(router.attributes.webspace)
+            .then(action((webspace) => {
+                this.webspace = webspace;
+            }));
+    }
+
+    componentWillUnmount() {
+        this.formStore.destroy();
+        const {resourceStore, router} = this.props;
+
+        if (resourceStore.locale) {
+            router.unbind('locale', resourceStore.locale);
+        }
+    }
+
+    handleSubmit = (options: ?Object) => {
+        const {resourceStore, router} = this.props;
+
+        const {
+            route: {
+                options: {
+                    editRoute,
+                },
+            },
+        } = router;
+
+        if (editRoute) {
+            resourceStore.destroy();
+        }
+
+        const saveOptions = {
+            webspace: router.attributes.webspace,
+            parent: router.attributes.parentId,
+            action: options ? options.action : undefined,
+        };
+
+        this.formStore.save(saveOptions)
+            .then(() => {
+                if (editRoute) {
+                    router.navigate(editRoute, {id: resourceStore.id, locale: resourceStore.locale});
+                }
+            });
+    };
+
+    setFormRef = (form) => {
+        this.form = form;
+    };
+
+    render() {
+        return (
+            <div className={pageEditStyles.form}>
+                <Form
+                    ref={this.setFormRef}
+                    store={this.formStore}
+                    onSubmit={this.handleSubmit}
+                />
+            </div>
+        );
+    }
+}
+
+export default withToolbar(PageEdit, function() {
+    const {router} = this.props;
+    const formTypes = this.formStore.types;
+    const webspace = this.webspace;
+
+    const backButton = {
+        onClick: () => {
+            const {resourceStore} = this.props;
+
+            const options = {};
+            options.locale = resourceStore.locale.get();
+            router.restore('sulu_content.webspaces', options);
+        },
+    };
+
+    const locale = webspace
+        ? {
+            value: this.props.resourceStore.locale.get(),
+            onChange: (locale) => {
+                this.props.resourceStore.setLocale(locale);
+            },
+            options: webspace.allLocalizations.map((localization) => ({
+                value: localization.localization,
+                label: localization.name,
+            })),
+        }
+        : undefined;
+
+    const items = [
+        {
+            type: 'dropdown',
+            label: translate('sulu_admin.save'),
+            icon: 'su-save',
+            loading: this.props.resourceStore.saving,
+            options: [
+                {
+                    label: translate('sulu_admin.save_draft'),
+                    disabled: !this.props.resourceStore.dirty,
+                    onClick: () => {
+                        this.form.submit({action: 'draft'});
```

Comment:
Replace with some other solution (either removing the `onSubmit` indirection or add multiple buttons, which could then be submitted like `this.form.submit('draft')`